### ISSUE TITLE
Relocate `plt.rcParams['figure.dpi'] = 150` in the demo notebook ?

### ISSUE THREAD

**Issue by GuillermoFidalgo** [masonproffitt/dance-at-snowmass#1]
This line in the first cell does not seem to be persistent after you create a histogram with `hist`. If you run the entire notebook once the dpi will remain to be 150 until you run
```python
h = hist.Hist(hist.axis.Regular(10, 0, 10, label='Number of muons in event'))
h.fill((branches['nMuon']))
h.plot()
plt.ylabel('Number of events')
plt.show()
```

If you print `plt.rcParams['figure.dpi']` you get 72.0

However it does work if you run the entire notebook twice.

**Comment by GuillermoFidalgo** [comment]
If you print `plt.rcParams['figure.dpi']` you get 72.0

**Comment by GuillermoFidalgo** [comment]
also 150 is quite large...

**Comment by masonproffitt** [comment]
Indeed, this line is an artifact from an older version that used only Matplotlib. It can probably just be removed now.
